src/pyseekdb/cli/__init__.py

```
@@ -1,14 +1,320 @@
+    except Exception:
+        info_text.append("Count: ", style="bold")
+        info_text.append("N/A\n", style="dim")
+
+    if coll.metadata:
+        info_text.append("Metadata: ", style="bold")
+        info_text.append(f"{coll.metadata}\n", style="dim")
+
+    return Panel(info_text, title=f"Collection: {coll.name}", border_style="blue")
+
+
+def _render_peek_table(records, count: int) -> Table:
```

Ruff (0.14.14)

[warning] Unused function argument: `count`

(ARG001)